setup.py

```diff
@@ -334,12 +356,12 @@ def run(self):
     },
     extras_require={
         # On Cloud TPU VM install with:
-        # $ sudo pip3 install torch_xla[tpuvm] -f https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-1.11-cp38-cp38-linux_x86_64.whl
+        # pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
```

Comment:
What's the difference between `pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html` and `pip install torch_xla[tpuvm]`? IOW, under what circumstance should I use the former versus the latter?

Reply:
The technical difference is that `tpuvm` refers to a private package repository directly. That's why you don't have to pass an `--extra-index`. PyPI disallows this, so with release wheels starting with 2.1, you have to use `tpu`, which specifies a libtpu version but not the location. The location comes from the index url.

I left `tpuvm` on master/nightly for now since there's no rush to remove it, but I think we might want to anyway to reduce confusion.

If you're not sure, just use the `tpu` extra dependency since we will have to support that going forward.

Reply:
Got it. Thanks!